Formula/s/screenpipe.rb

```diff
@@ -0,0 +1,38 @@
+class Screenpipe < Formula
+  desc "Library to build personalized AI powered by what you've seen, said, or heard"
+  homepage "https://github.com/mediar-ai/screenpipe"
+  url "https://github.com/mediar-ai/screenpipe/archive/refs/tags/v0.1.98.tar.gz"
+  sha256 "cb3c8039ecb60d35bacd2b9673db112f907b4a1d3d7c32f49a5e77c0274268ad"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "rust" => :build
+  depends_on "ffmpeg"
+  depends_on :macos
```

Comment:
What makes this macOS-only?

Reply:
I don't think building on Linux is disobedience. In any case, projects releasing binaries only for specific OSes doesn't imply a dependency on that particular OS.